src/admin/utilities/getSupportedMonacoLocale.ts

```diff
@@ -0,0 +1,15 @@
+export const getSupportedMonacoLocale = (locale: string): string => {
+  const supportedLocales = {
```

Comment:
Where did you find this list of supported lanagues so we can maintain it in the future?